guides/common/modules/con_monitoring-hosts-by-using-insights-in-cloud.adoc

```diff
@@ -0,0 +1,39 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="monitoring-hosts-by-using-{insights-id}-in-cloud"]
+= Monitoring hosts by using {Insights} in {RHCloud}
+
+[role="_abstract"]
+You can use the {Insights} advisor service in the {RHCloud} to assess system health on hosts running {RHEL}.
+You can use the {Insights} dashboard in the {RHCloud} to quickly analyze recommendations for remediation, vulnerabilities, and more.
+
+If you use {Insights} in the {RHCloud}:
+
+* You can use {Insights} services online in the {RHCloud}.
+* The Insights client collects data of your hosts and {Project} forwards the data through the Insights client to the {RHCloud} to calculate recommendations.
+* You can view and remediate those recommendations both in the {ProjectWebUI} and in the {RHCloud}.
+* {Project} uploads host inventory reports to the {RHCloud}.
+* You have control over how much information is included in those reports.
+* You cannot use local {insights-iop}.
+
+.Prerequisites
+* For new {Project} hosts, install and configure the Insights client during host registration to {Project}.
```

Comment:
I do not like "{Project} hosts". It's unclear if that is any Smart Proxy Server or any host. Reading on, it's clear that it's both technically. I suggest to reword it:

```suggestion
* Install and configure the Insights client during host registration to {Project} in your registration and provisioning workflows.
```

AFAIK each supported provisioning workflow registers the host to Foreman+Katello too. The alternative is to split this into three parts:

* Registering existing hosts (this line+21)
* Provisioning hosts (kinda missing/not that explicit)
* Enabling on hosts via Ansible (see line 22)